### DISCUSSION TITLE
Free Trial Users: Grid Date Time Formating

### DISCUSSION THREAD

**Discussion by surtic86** [Glench/ExtPay#38] [General]
Hi there,

It would be nice when the Date Time in the Free Trial Users Grid would be a bit more formatted, shorten like YYYY-MM-DD HH:MM without all the Timezone hassle.

Cheers 
surtic86

**Comment by burkybang** [comment]
I have a personal extension that I use for myself on all websites. I have a content script on extensionpay.com. Your request is one of the things it does.
